gz-resources.html

```diff
@@ -468,6 +468,10 @@
           }
         }
         console.error(msg)
```

Comment:
ah that's true, this is on the browser. Looks good the way it is. I'll merge this and make a new release ;)